src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -64,4 +64,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("3.5.0.6885")]
+[assembly: AssemblyFileVersion("3.5.0.7533")]
```

Comment:
hi @johnpierson Just a tip, Dynamo team usually do not commit the change automatically generated in this file to avoid merge conflicts in PRs because when everyone modifies this, for sure there will be conflicts :)

Reply:
Thank you!